sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2998,6 +2998,14 @@ SIREPO.app.directive('simStatusPanel', function(appState) {
                     || 'Simulation ' + $scope.simState.stateAsText() + ': ' + $scope.simState.getFrameCount() + ' animation frames';
             };
             $scope.start = function() {
+                // The available jobRunModes can change. Default to parallel if
+                // the current jobRunMode doesn't exist
+                if (
+                    appState.models[$scope.simState.model].jobRunMode
+                        in authState.jobRunModeMap === false
```

Comment:
We do not want to set jobRunMode on a model if it doesn't exist so the condition needs to be changed to:

```js
var j = appState.models[$scope.simState.model].jobRunMode;
if (j && j in authState.jobRunModeMap === false)
```

Just a note to JS novices like me: `undefined in {}` is a valid test, which I find interesting. Moreover, `! undefined in {} === false` and `undefined in {sbatch: 'sbatch'} === false` are both true. `!` has higher precedence than `in`.  I see why the `=== false` is necessary, but it's weird.